nettacker/modules/vuln/crushftp_cve_2025_31161.yaml

```
@@ -0,0 +1,58 @@
+                - 80
+                - 443
+                - 8080
+        response:
+          condition_type: and
+          conditions:
+            status_code:
+              regex: "200"
+              reverse: false
+            content:
+              regex: '<user_list_subitem>crushadmin</user_list_subitem>'
+              reverse: false
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)